src/main/java/com/flightstats/hub/webhook/strategy/TimedWebhookStrategy.java

```diff
@@ -244,6 +270,13 @@ private void doWork() throws InterruptedException {
         return Optional.ofNullable(queue.poll(10, TimeUnit.MINUTES));
     }
 
+    @VisibleForTesting
+    SortedSet<ContentPathKeys> drainQueue() {
```

Comment:
This allows the test to interrogate what was added to the queue after running `getContentKeyGenerator`